config/config.go

```diff
@@ -0,0 +1,142 @@
+package config
+
+import (
+	"github.com/nhost/be/services/mimir/model"
+	"github.com/nhost/be/services/mimir/schema"
+	"github.com/nhost/cli/internal/generichelper"
+	"github.com/nhost/cli/internal/ports"
+	"github.com/nhost/cli/util"
+	"github.com/pelletier/go-toml/v2"
+)
+
+type Config struct {
```

Comment:
Why the wrapper? Doesn't seem to introduce any functionality or benefits.

Reply:
Removed